CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 10.0.0-rc.1 "paper-person" (2020-06-08)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **testing/testbed:** set defined event properties as configurable ([94571c5](https://github.com/angular/components/commit/94571c5)) |
+| bug fix |  **testing/testbed:** synthetic DOM events cannot be prevented multiple times ([bde3af8](https://github.com/angular/components/commit/bde3af8)), closes [#19440](https://github.com/angular/components/issues/19440) |
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **checkbox:** high contrast focus indication not working ([#19488](https://github.com/angular/components/issues/19488)) ([a95da92](https://github.com/angular/components/commit/a95da92)), closes [#19443](https://github.com/angular/components/issues/19443) |
+| bug fix |  **checkbox:** pass aria-describedby through to input ([#19495](https://github.com/angular/components/issues/19495)) ([e1d85d2](https://github.com/angular/components/commit/e1d85d2)), closes [#19477](https://github.com/angular/components/issues/19477) |
+| bug fix |  **datepicker:** compilation errors with ViewEngine ([#19516](https://github.com/angular/components/issues/19516)) ([00e2171](https://github.com/angular/components/commit/00e2171)) |
+| bug fix |  Focus indicators are now positioned ([#19345](https://github.com/angular/components/issues/19345)) ([c05a07e](https://github.com/angular/components/commit/c05a07e)) |
+| bug fix |  **overlay:** resolve circular dependency ([#19433](https://github.com/angular/components/issues/19433)) ([7037c3a](https://github.com/angular/components/commit/7037c3a)) |
```

Comment:
I think this is not really a `bug fix` as this is just type-only circular dependency. Surely can just leave it as is 😄